torch_geometric/nn/conv/ssg_conv.py

```diff
@@ -0,0 +1,118 @@
+from typing import Optional
+
+from torch import Tensor
+from torch_sparse import SparseTensor, matmul
+
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.nn.conv.gcn_conv import gcn_norm
+from torch_geometric.nn.dense.linear import Linear
+from torch_geometric.typing import Adj, OptTensor
+
+
+class SSGConv(MessagePassing):
+    r"""The simple spectral graph convolutional operator from the
+    `"Simple Spectral Graph Convolution"
+    <https://openreview.net/forum?id=CYO5T-YjWZV>`_ paper
+
+    .. math::
+        \mathbf{X}^{\prime} = \frac{1}{K} \sum_{k=1}^K\left((1-\alpha)
+        {\left(\mathbf{\hat{D}}^{-1/2} \mathbf{\hat{A}}
+        \mathbf{\hat{D}}^{-1/2} \right)}^k
+        \mathbf{X}+\alpha \mathbf{X}\right) \mathbf{\Theta},
+
+    where :math:`\mathbf{\hat{A}} = \mathbf{A} + \mathbf{I}` denotes the
+    adjacency matrix with inserted self-loops and
+    :math:`\hat{D}_{ii} = \sum_{j=0} \hat{A}_{ij}` its diagonal degree matrix.
+    The adjacency matrix can include other values than :obj:`1` representing
+    edge weights via the optional :obj:`edge_weight` tensor.
```

Comment:
Maybe point out the relationship to `SGConv` and the additional `alpha` in the doc-string.